src/main/kotlin/uk/gov/justice/hmpps/architecture/model/HMPPSDataAPI.kt

```diff
@@ -0,0 +1,98 @@
+package uk.gov.justice.hmpps.architecture.model
+
+import com.structurizr.model.Model
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.HMPPSSoftwareSystem
+
+class HMPPSDataAPI private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+
+    override fun defineModelEntities(model: Model) {
+
+      system = model.addSoftwareSystem(
+        "HMPPS Data APIs",
+        "Internal Data API Systems"
+      )
+    }
+
+    override fun defineRelationships() {
+    }
+
+    override fun defineViews(views: ViewSet) {
+
+      views.createContainerView(
+        system,
```

Comment:
This sounds like a grouping which doesn't _need_ a bespoke system (all systems show up on the landscape diagram, so this would create "dangling" ones)

I wonder if we could use https://c4model.com/#SystemLandscapeDiagram instead?

Reply:
I tried a system landscape initially but some of the API services are modelled as containers which can't be included. It's probably worth promoting these to systems over time which would solve the problem. I agree these views are not systems so I'll try and come up with another solution

Reply:
Cool, thanks. I guess let's leave them in for now then